utils/report.c

```diff
@@ -82,6 +84,96 @@ static struct uftrace_report_node *find_or_create_node(struct rb_root *root, con
 	return node;
 }
 
+static struct uftrace_report_call *find_or_create_report_call(struct uftrace_fstack *fstack)
+{
+	struct uftrace_report_call *report_call;
+
+	list_for_each_entry(report_call, &report_calls, list) {
+		if (report_call->addr == fstack->addr)
+			return report_call;
```

Comment:
A linear search?  I think we need to use a rbtree for scalability.

Reply:
I changed it to the rbtree.